src/test/scala/io/citrine/lolo/bags/BaggerTest.scala

```diff
@@ -77,6 +77,35 @@ class BaggerTest {
   }
 
   /**
+    * Test that the uncertainty metrics are properly calibrated
+    *
+    * This test is based on the "standard" RMSE, which is computed by dividing the error
+    * by the predicted uncertainty.  On the exterior, these is an additional extrapolative
+    * uncertainty that isn't captured well by this method, so we test the interior and the full
+    * set independently
+    */
+  def testUncertaintyCalibration(): Unit = {
+    val width = 0.1
+    val trainingData = TestUtils.generateTrainingData(128, 6, xscale = width, seed = 6L)
+    val DTLearner = new RegressionTreeLearner(numFeatures = 6)
+    val bias = new RegressionTreeLearner(maxDepth = 4)
+    val baggedLearner = new Bagger(DTLearner, numBags = 16 * trainingData.size, biasLearner = Some(bias))
+    val RFMeta = baggedLearner.train(trainingData)
+    val RF = RFMeta.getModel()
+
+    val interiorTestSet = TestUtils.generateTrainingData(64, 6, xscale = width/2.0, xoff = width/4.0, seed = 7L)
+    val fullTestSet = TestUtils.generateTrainingData(64, 6, xscale = width, seed = 7L)
+
+    val interiorStandardRMSE = BaggerTest.getStandardRMSE(interiorTestSet, RF)
+    val fullStandardRMSE = BaggerTest.getStandardRMSE(fullTestSet, RF)
+    assert(interiorStandardRMSE > 0.85, "Standard RMSE in the interior should be greater than 0.85")
+    assert(interiorStandardRMSE < 1.0, "Standard RMSE in the interior should be less than 1.0")
+
```

Comment:
With what probability should these assert statements be true if you don't fix the random seeds?